incubator/orientdb/templates/statefulset.yaml

```diff
@@ -122,23 +124,41 @@ spec:
             name: {{ template "orientdb.fullname" . }}-configmap
         - name: orientdb-config-vol
           emptyDir: {}
+  {{- if (.Values.persistence.enabled) and (.Values.persistence.volumeClaim) }}
+      {{- if .Values.persistence.volumeClaim.storageName }}
```

Comment:
template: orientdb/templates/statefulset.yaml:128:20: executing "orientdb/templates/statefulset.yaml" at <.Values.persistence....>: can't evaluate field storageName in type interface {}

complained by circleci-chart-lint

Reply:
I have extended the same and may not be needed for everyone with existing volumeclaim, so have commented .Values.persistence.volumeClaim.storageName in values.yaml file. However if line 127 is not satisfied, it should get into the 128th line. Correct me if I am wrong here.

Reply:
Line 127 itself basically is wrong. It should be 

'''
{{- if and (.Values.persistence.enabled) (.Values.persistence.volumeClaim) }}
'''

Reply:
Got It! I will fix it properly and resubmit